### PR TITLE
Libs update

### PR DESCRIPTION
**Pull request by vterekhov81**
* All jar-files of libs were removed.
* Now the Project are under Gradle.
* Most local jars-files of libs changed to Gradle dependencies to repositories. The project has no errors and has only 2 warnings during building.

### CODE REVIEW

**Comment by vterekhov81** [comment]
* The TestFlight was replaced by the HockeyApp.

**Comment by fo2rist** [comment]
new comment

File: AndroidSuperProject/.classpath [hunk 1]
```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="src" path="gen"/>
 	<classpathentry kind="con" path="com.android.ide.eclipse.adt.ANDROID_FRAMEWORK"/>
 	<classpathentry exported="true" kind="con" path="com.android.ide.eclipse.adt.LIBRARIES"/>
 	<classpathentry exported="true" kind="con" path="com.android.ide.eclipse.adt.DEPENDENCIES"/>
 	<classpathentry kind="lib" path="libs"/>
+	<classpathentry kind="lib" path="libs/acra-4.5.0.jar"/>
```
Comment (fo2rist) [inline AndroidSuperProject/.classpath]: Думаю, это нам тоже не нужно с хоккеем.

File: AndroidSuperProject/build.gradle [hunk 2]
```diff
@@ -0,0 +1,55 @@
+apply plugin: 'android'
+
+dependencies {
+    compile fileTree(include: '*.jar', dir: 'libs')
+    compile project(':facebook-libs-ver-3.8')
+    compile ('eu.inmite.android.lib:android-validation-komensky:0.9.2')
+    {
+        exclude module: 'support-v4'
+    }
+//    compile 'ch.acra:acra:4.5.0'
```
Comment (fo2rist) [inline AndroidSuperProject/build.gradle]: Just remove it

File: AndroidSuperProject/res/values/strings_activity_login.xml [hunk 3]
```diff
@@ -1,9 +1,7 @@
 <resources>
-    <!-- Strings related to login -->
-    <string name="action_sign_in">Sign in</string>
+    <!-- Strings related to login -->
+    <string name="action_sign_in">Sign in</string>
     <string name="action_create_account">Create an account</string>
     <string name="action_sign_in_with_social_network">Sign in with social network</string>
     <string name="action_forgot_password">Recover password</string>
-    <string name="action_sign_in_with_facebook">Sign in with facebook</string>
```
Comment (fo2rist) [inline AndroidSuperProject/res/values/strings_activity_login.xml]: Do we really don't need it anymore?

File: AndroidSuperProject/src/com/ruswizards/androidsuperproject/Constants.java [hunk 4]
```diff
@@ -3,6 +3,8 @@
 public class Constants {
 
 	/** Tag for logging. Should be changed */
-	public static final String TAG = "AndroidSuperProject";
+	public static final String TAG = "SuperProj";
```
Comment (fo2rist) [inline AndroidSuperProject/src/com/ruswizards/androidsuperproject/Constants.java]: Why do we need this change?

File: AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/LoginActivity.java [hunk 5]
```diff
@@ -73,18 +75,20 @@ protected void onCreate(Bundle savedInstanceState) {
...
-	@Override
-	protected void onPause() {
-		FormValidator.stopLiveValidation(this);
-		super.onPause();
+	}
+
+	@Override
+	protected void onResume() {
+		super.onResume();
+        CrashManager.register(this, Constants.HOCKEY_APP_ID);
```
Comment (fo2rist) [inline AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/LoginActivity.java]: bad indent

File: AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/LoginActivity.java [hunk 6]
```diff
@@ -190,6 +194,11 @@ private void onCreateAccountClicked(View sender) {
 	 */
 	@InjectOnClickListener(R.id.social_sign_in)
 	private void onLoginWithSocialNetworkClicked(View sender) {
+
+
+        int ee=1/0;
```
Comment (fo2rist) [inline AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/LoginActivity.java]: Forgotten crash code. Remove it

File: AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java [hunk 7]
```diff
@@ -15,6 +20,10 @@
  */
 public class SplashActivity extends Activity {
 
+    private static final String TAG = Constants.TAG;// + " - " + SplashActivity.class.getName();
```
Comment (fo2rist) [inline AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java]: remove unused code

File: AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java [hunk 8]
```diff
@@ -56,10 +65,51 @@ protected void onCreate(Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
 		setContentView(R.layout.activity_splash);
 
+        Log.e(TAG, "onCreate");
+
+//        crashManagerListener = new CrashManagerListener() {
```
Comment (fo2rist) [inline AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java]: Unused code

File: AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java [hunk 9]
```diff
@@ -56,10 +65,51 @@ protected void onCreate(Bundle savedInstanceState) {
...
 		timerTimeTask_.execute();
 	}
 
+    @Override
+    public void onResume()
+    {
+        super.onResume();
+        Log.e(TAG, "onResume");
+
+        //http://support.hockeyapp.net/discussions/problems/29251-crash-report-not-getting-logged-in
```
Comment (fo2rist) [inline AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java]: Same

File: AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java [hunk 10]
```diff
@@ -56,10 +65,51 @@ protected void onCreate(Bundle savedInstanceState) {
...
+//        CrashManager.execute(getApplicationContext(), crashManagerListener);
+    }
+
+    @Override
+    public void onPause()
+    {
+        super.onPause();
```
Comment (fo2rist) [inline AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java]: We don't need it

File: AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java [hunk 11]
```diff
@@ -56,10 +65,51 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        Log.e(TAG, "onPause");
+    }
+
+    @Override
+    public void onDestroy()
+    {
+        super.onDestroy();
+        Log.e(TAG, "onDestroy");
```
Comment (fo2rist) [inline AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java]: Same

File: AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java [hunk 12]
```diff
@@ -56,10 +65,51 @@ protected void onCreate(Bundle savedInstanceState) {
...
+
 		timerTimeTask_ = new StopSplashTask();
 		timerTimeTask_.execute();
 	}
 
+    @Override
+    public void onResume()
+    {
+        super.onResume();
+        Log.e(TAG, "onResume");
```
Comment (fo2rist) [inline AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java]: Unnecessary logging

File: AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java [hunk 13]
```diff
@@ -45,8 +54,8 @@ protected void onPostExecute(Void result) {
 				startActivity(LoginActivity.createIntent(SplashActivity.this));
 			}
 			finish();
+            Log.e(TAG, "finish");
```
Comment (fo2rist) [inline AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java]: Unnecessary logging

**Comment by oxied** [comment]
Need to fix gradle files. Vladimir, you know the problem

screenshot 2015-04-01 00 27 57

File: AndroidSuperProject/res/values/strings_activity_login.xml (hunk 3 above)
Comment (0neel) [inline AndroidSuperProject/res/values/strings_activity_login.xml]: No, we don't. This string is used in social sign in activity.

File: AndroidSuperProject/.classpath (hunk 1 above)
Comment (0neel) [inline AndroidSuperProject/.classpath]: Deleted

File: AndroidSuperProject/build.gradle (hunk 2 above)
Comment (0neel) [inline AndroidSuperProject/build.gradle]: Removed

File: AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/LoginActivity.java (hunk 5 above)
Comment (0neel) [inline AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/LoginActivity.java]: Fixed

File: AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/LoginActivity.java (hunk 6 above)
Comment (0neel) [inline AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/LoginActivity.java]: Removed

File: AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java (hunk 7 above)
Comment (0neel) [inline AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java]: Removed

File: AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java (hunk 8 above)
Comment (0neel) [inline AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java]: Removed

File: AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java (hunk 9 above)
Comment (0neel) [inline AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java]: Removed

File: AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java (hunk 10 above)
Comment (0neel) [inline AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java]: Removed

File: AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java (hunk 11 above)
Comment (0neel) [inline AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java]: Removed

File: AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java (hunk 12 above)
Comment (0neel) [inline AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java]: Removed

File: AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java (hunk 13 above)
Comment (0neel) [inline AndroidSuperProject/src/com/ruswizards/androidsuperproject/activities/SplashActivity.java]: Removed

File: AndroidSuperProject/src/com/ruswizards/androidsuperproject/Constants.java (hunk 4 above)
Comment (0neel) [inline AndroidSuperProject/src/com/ruswizards/androidsuperproject/Constants.java]: Reverted

**Comment by oxied** [comment]
Good.

**Pull request merged**